src/Kafka/KafkaMessageBrowser/components/LimitSelector.tsx

```diff
@@ -40,6 +40,7 @@ export const LimitSelector: VoidFunctionComponent<LimitSelectorProps> = ({
           isOpen={isOpen}
           isDisabled={isDisabled}
           onSelect={() => setIsOpen(false)}
+          data-testid={"limit-selector"}
```

Comment:
For some reason, this ID is not on the div that I would need it to be, but on <ul> element when you expand the select component.
![image](https://user-images.githubusercontent.com/20477661/185880985-8861a087-0db6-4934-a6c9-f4ca9ab08217.png)

Reply:
That's unfortunately how the PF select handles the attribute, we can't do much about it I think.

Reply:
In that case, I will use `/parent::div` in the selector then... Thanks!